gokrazy.go

```diff
@@ -118,6 +121,38 @@ func Boot(userBuildTimestamp string) error {
 
 	initRemoteSyslog()
 
+	if _, err := os.Stat("/etc/ssl/web.pem"); !os.IsNotExist(err) {
+		cert, err := tls.LoadX509KeyPair("/etc/ssl/web.pem", "/etc/ssl/web_key.pem")
+		if err != nil {
+			return fmt.Errorf("failed loading certificate: %v", err)
+		} else {
+			useTLS = true
+			tlsConfig = &tls.Config{
+				Certificates: []tls.Certificate{cert},
+				// Let's be paranoid!
```

Comment:
Should we follow the recommendations on https://cipherlist.eu/ for a Go server instead? (and reference that in the comment so that it can be updated)